tests/storage/outofprocess_test.py

```diff
@@ -967,61 +975,55 @@ def test_glob(oop_cleanup, tmpdir):
 
 # External APIs
 
-def test_read_lines(oop_cleanup, tmpdir):
+def test_read_lines(oop_cleanup, usr_file_512):
     iop = oop.getProcessPool("test")
-    path = str(tmpdir.join("file"))
 
-    with open(path, "wb") as f:
+    with open(usr_file_512, "wb") as f:
         f.write(b"1\n2\n3\n")
-    assert iop.readLines(path) == [b"1", b"2", b"3"]
+    assert iop.readLines(usr_file_512) == [b"1", b"2", b"3"]
```

Comment:
Looks fine, and will be better when using a more generic name (user_file).

Reply:
Addressed in another PR after splitting commits (ended up using user_mount).